experiment-rollout.js

```diff
@@ -46,33 +46,45 @@ const data = {
         priority: [
             {
                 status: 2,
-                name: 'Brands'
+                name: 'Brands | Reserved unless Discord partner'
             },
             {
                 status: 2,
-                name: 'Discord Staff'
+                name: 'Large bots | Criteria unknown | Reservations only'
             },
             {
                 status: 2,
-                name: 'Partnered, subscription-enabled (100USD/mo+), and verified server owners (includes some old partners + some content creators)'
+                name: 'Discord Staff | Staff and personal accounts'
             },
             {
-                status: 1,
-                name: 'Verified bots (Server count wins name conflicts, other ones will get their discriminator appended)'
+                status: 2,
+                name: 'Partnered / verified / ex-partnered server owners'
             },
             {
-                status: 0,
-                name: 'Top (likely 100, by server count) bot developers (Unconfirmed)'
+                status: 1,
+                name: 'Verified bots | Ordered by support server member count'
```

Comment:
where did you get that "support server member count" info?

Reply:
Thats from the "Server count wins name conflicts, other ones will get their discriminator appended", reworded

Reply:
I misinterpreted the original priority name - fixed